x/staking/autocli.go

```diff
@@ -142,6 +143,57 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 		},
 		Tx: &autocliv1.ServiceCommandDescriptor{
 			Service: stakingv1beta.Msg_ServiceDesc.ServiceName,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod:      "Delegate",
+					Use:            "delegate [validator-addr] [amount] --from [delegator_address]",
+					Short:          "Delegate liquid tokens to a validator",
+					Long:           "Delegate an amount of liquid coins to a validator from your wallet.",
+					Example:        fmt.Sprintf("%s tx staking delegate cosmosvalopers1l2rsakp388kuv9k8qzq6lrm9taddae7fpx59wm 1000stake --from mykey", version.AppName),
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "validator_address"}, {ProtoField: "amount"}},
+				},
+				{
+					RpcMethod: "BeginRedelegate",
+					Use:       "redelegate [src-validator-addr] [dst-validator-addr] [amount] --from [delegator]",
+					Short:     "Generate multisig signatures for transactions generated offline",
+					Long: `Sign transactions created with the --generate-only flag that require multisig signatures.
+
+Read one or more signatures from one or more [signature] file, generate a multisig signature compliant to the
+multisig key [name], and attach the key name to the transaction read from [file].
+
+If --signature-only flag is on, output a JSON representation
+of only the generated signature.
+
+If the --offline flag is on, the client will not reach out to an external node.
+Account number or sequence number lookups are not performed so you must
+set these parameters manually.
+
+The current multisig implementation defaults to amino-json sign mode.
+The SIGN_MODE_DIRECT sign mode is not supported.`,
+					Example:        fmt.Sprintf(`%s tx multisign transaction.json k1k2k3 k1sig.json k2sig.json k3sig.json`, version.AppName),
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "validator_src_address"}, {ProtoField: "validator_dst_address"}, {ProtoField: "amount"}},
+				},
+				{
+					RpcMethod:      "Undelegate",
+					Use:            "unbond [validator-addr] [amount] --from [delegator_address]",
+					Short:          "Unbond shares from a validator",
+					Long:           "Unbond an amount of bonded shares from a validator.",
+					Example:        fmt.Sprintf(`%s tx staking unbond %s1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj 100stake --from mykey`, version.AppName, sdk.GetConfig().GetBech32ValidatorAddrPrefix()),
```

Comment:
Let's not introduce usage of sdk config. Let's just say `[validator-addr]` instead